pandokia/import_data.py

```diff
@@ -278,9 +284,26 @@ def try_insert(self, db, key_id):
                  self.attn,
                  self.test_runner,
                  okf]
+        
+        hash_str = self.test_run+self.host+self.project+self.test_name+self.context
+
+        # return a string of length 32
+        h = hashlib.md5(hash_str.encode())
```

Comment:
Did you verify these lines are the same in py2 and py3?  If not, you can simply play with md5 calls in two different interpreter windows in your terminal and make sure they take/return the types you are expecting that they do.

Reply:
Yes, I have tested the hashlib functions in both python 2 and 3, they both work as expected.